tests/test_cli.py

```diff
@@ -238,11 +245,71 @@ def test_run_with_pattern(cli: click.testing.CliRunner) -> None:
                 ]
             )
             assert kwargs["pattern"].pattern == "^pattern.*"
+            assert kwargs["venv_pattern"].pattern == ".*"
             assert kwargs["recreate_venvs"] is False
             assert kwargs["skip_base_install"] is False
             assert kwargs["pass_env"] is False
 
 
+def test_run_no_venv_pattern(cli: click.testing.CliRunner) -> None:
+    """Running run with pattern passes in that pattern."""
+    with mock.patch("riot.riot.logger.debug") as mock_debug:
+        with with_riotfile(cli, "simple_riotfile.py"):
+            result = cli.invoke(
+                riot.cli.main,
+                [
+                    "run",
+                    "test",
+                    "-d",
+                    "--skip-base-install",
+                ],
+            )
+            # Success, but no output because we mock run
+            assert result.exit_code == 0
+            assert result.stdout == ""
+
+            mock_debug.assert_called()
+            assert not any(
```

Comment:
Ideally this should be replaced with checking the `result.stdout` above. Once we move to using click.echo for program printing we should be able to change these over.